build.sbt

```diff
@@ -1,73 +1,22 @@
 import sbt.Path.relativeTo
 
-ThisBuild / majorVersion := 1
-ThisBuild / isPublicArtefact := true
-
-val scala2_12 = "2.12.18"
 val scala2_13 = "2.13.12"
 
-def copySources(module: Project) = Seq(
-  Compile / scalaSource := (module / Compile / scalaSource).value,
-  Compile / resourceDirectory := (module / Compile / resourceDirectory).value,
-  Test / scalaSource := (module / Test / scalaSource).value,
-  Test / resourceDirectory := (module / Test / resourceDirectory).value
-)
-
-lazy val library = (project in file("."))
-  .settings(publish / skip := true)
```

Comment:
not copied this to playNunjucksPlay30 because we do want to publish the "play-nunjucks-play-30" project

Reply:
@oscarduignan Does this mean the artefact name will change from `play-nunjucks-play-30` back to `play-nunjucks`?

Reply:
it stays as play-nunjucks-play-30, because I've left the play nunjucks project as
```
Project("play-nunjucks-play-30", file("play-nunjucks-play-30"))
```
so it has the [id](https://github.com/sbt/sbt/blob/1.10.x/main/src/main/scala/sbt/Project.scala#L172) explicitly defined, rather than using [the project macro which uses the name of the val as the id](https://github.com/sbt/sbt/blob/1.10.x/main/src/main/scala/sbt/Project.scala#L998)

possibly still be annoying for future maintainers to have the folder have the version number in it, probably be better to have just play-nunjucks so never have to rename it, or hoist the src dir up again to top level, but those are big enough changes that I would do them later and probably leave it up to future maintainers

Reply:
have renamed the folders now but kept it a separate commit to make it easier to review just the changes